adapter/CCXHelpers.h

```diff
@@ -53,6 +53,17 @@ enum CouplingDataType { TEMPERATURE,
                         VELOCITIES,
                         POSITIONS };
 
+/**
+ * @brief Type of element used for faces mesh, where we assume only one type of element is used.
+ *  TETRAHEDRA      - C3D4 or C3D10 element
+ *  HEXAEDRA        - C3D8 or C3D20 elment
+ *  INVALID_ELEMENT - Anything else
+ */
+enum ElemType { TETRAHEDRA,
+                HEXAEDRA,
```

Comment:
```suggestion
                HEXAHEDRA,
```

Reply:
@boris-martin this change is still not included, and hence I cannot compile the adapter for now, to test the heat-exchanger case.

Reply:
My bad! Done